tests/dummy/app/templates/components/print-test-attributes.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="attr">{{data-test-attribute}}</div>
```

Comment:
This is an example of using a `data-test-*` attribute in the wrong place actually. It should be stripped but should actually never appear here in the first place.